compliance_checker/base.py

```diff
@@ -123,7 +123,11 @@ def __init__(self,
         if value is None:
             self.value = None
         elif isinstance(value, tuple):
-            assert len(value) == 2, 'Result value must be 2-tuple or boolean!'
+            try:
+                assert len(value) == 2, 'Result value must be 2-tuple or boolean!'
+            except AssertionError:
+                if value == (): # just make it (0, 0)
+                    value = (0, 0)
```

Comment:
When does an empty tuple get passed in? Was this something new that was introduced in this PR or is this fixing something that had always been an issue?